src/components/Create/components/pages/NftRewards/hooks/NftRewardsForm.ts

```diff
@@ -38,11 +38,22 @@ export const useNftRewardsForm = () => {
       imgUrl: t.imageUrl,
     }))
 
-    return { rewards, collectionName, collectionSymbol, collectionDescription }
+    return {
+      rewards,
+      collectionName,
+      collectionSymbol,
+      collectionDescription,
+      postPayMessage: postPayModal?.content,
+      postPayButtonText: postPayModal?.ctaText,
+      postPayButtonLink: withoutHttp(postPayModal?.ctaLink),
```

Comment:
**suggestion** can't we just make any link always include the https prefix? Confusing to have both options

Reply:
wdym?